### PR TITLE
Adjusts Chaplain Special Bible Equipment

### PR DESCRIPTION
**Pull request by Cheridan**
> Get a free clown mask for picking a certain icon
> Also get TWO banana peels for free when the _ACTUAL FREAKING CLOWN_ gets just one
> Get null rod, bible healing, incinerator/mass driver on top of that

Don't even act mad at this PR because you probably didn't know this existed!

### CODE REVIEW

File: code/game/objects/items/weapons/storage/book.dm [hunk 1]
```diff
@@ -68,11 +68,8 @@ var/global/list/bibleitemstates =	list("bible", "koran", "scrapbook", "bible", "
 /obj/item/weapon/storage/book/bible/proc/setupbiblespecifics(var/obj/item/weapon/storage/book/bible/B, var/mob/living/carbon/human/H)
 	switch(B.icon_state)
 		if("honk1","honk2")
-			new /obj/item/weapon/grown/bananapeel(B)
-			new /obj/item/weapon/grown/bananapeel(B)
-
-			if(B.icon_state == "honk1")
-				H.equip_to_slot_or_del(new /obj/item/clothing/mask/gas/clown_hat(H), slot_wear_mask)
+			new /obj/item/weapon/bikehorn(B)
+			H.dna.add_mutation(CLOWNMUT)
```
Comment (optimumtact) [inline code/game/objects/items/weapons/storage/book.dm]: :sweat:

Comment (Cheridan) [inline code/game/objects/items/weapons/storage/book.dm]: If you wanna run a clown gimmick then you can go all the way with it tbh.

**Comment by KorPhaeron** [comment]
Time for a new codebase

File: code/game/objects/items/weapons/storage/book.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/objects/items/weapons/storage/book.dm]: WOW THIS RUINS THE ENTIRE GAME EVERYWHERE FOR EVERYONE

**Comment by KorPhaeron** [comment]
In all seriousness though he should probably keep the mask if praying to clowns is part of his gimmick

**Comment by Cheridan** [comment]
fine

**Comment by ike709** [comment]
I actually did know about this and I did actually chose to worship the honk gods almost everytime I played chaplain... Honk... :-1: 





:+1:

File: code/game/objects/items/weapons/storage/book.dm (hunk 1 above)
Comment (Firecage) [inline code/game/objects/items/weapons/storage/book.dm]: @Cheridan But whyyyyy. THis didn't hurt anyone. Re-add clown mask!

**Comment by Supermichael777** [comment]
Well less free stuns is good. not sure about clown mutation might want to antag check it. clown-mode can realy fuck you over if you don't plan around it.

**Comment by HoaxIT** [comment]
WOW! TWO BANANA PEALS, THE WORLD IS ENDING.

**Comment by Dorsisdwarf** [comment]
What Hoax said

**Comment by Durkel** [comment]
Holy shit two banana peels. The power creep is real.

File: code/game/objects/items/weapons/storage/book.dm (hunk 1 above)
Comment (Cheridan) [inline code/game/objects/items/weapons/storage/book.dm]: i already did before you even commented 8^I

**Comment by SconesC** [comment]
Holy fuck the bottle knock into clown mut injector is gonna be reaaaaal

**Comment by freerealestate** [comment]
I don't know why you would ever do this

**Comment by bgobandit** [comment]
I actually did know about this. All hail the Honkmother. This PR is heretic and the work of mimes

**Comment by mrpain666** [comment]
When I played here I would run a "priest of the Honkmother" gimmick as chaplain, draw a yellow crayon rune, sacrifice my banana peels on them to the Honkmother and ask for a bike horn in return.

Not really seeing the big deal about removing banana peels, its not like I couldnt easily get more from botany who was already very likely growing very large quantities of them before I even thought to ask for them.

**Comment by RemieRichards** [comment]
Honk

**Pull request merged**